lib/srv/desktop/rdp/rdpclient/src/client.rs

```diff
@@ -463,14 +472,11 @@ pub type FunctionReceiver = Receiver<ClientFunction>;
 type RdpReadStream = Framed<TokioStream<ReadHalf<TlsStream<TokioTcpStream>>>>;
 type RdpWriteStream = Framed<TokioStream<WriteHalf<TlsStream<TokioTcpStream>>>>;
 
-fn create_config(params: ConnectParams) -> Config {
+fn create_config(width: u16, height: u16, username: String) -> Config {
     Config {
-        desktop_size: ironrdp_connector::DesktopSize {
-            width: params.screen_width,
-            height: params.screen_height,
-        },
+        desktop_size: ironrdp_connector::DesktopSize { width, height },
         security_protocol: SecurityProtocol::HYBRID_EX,
-        username: params.username,
+        username,
         password: std::env::var("RDP_PASSWORD").unwrap(), //todo(isaiah)
```

Comment:
Should we remove `password` or are we not ready for it yet?

Reply:
We could work on that at this point, though it would break the overall login (since there's a lot of PDUs that aren't implemented yet). Currently I'm just focused on porting more PDUs, and plan to get to that once that broader task is complete.